ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -334,8 +352,11 @@ sub restart_opennebula_service {
         $srv = CAF::Service->new(['opennebula'], log => $self);
     } elsif ($service eq "sunstone") {
         $srv = CAF::Service->new(['opennebula-sunstone'], log => $self);
+    } elsif ($service eq "kvmrc") {
+        $self->info("Updated $service file. onehost sync is required.");
+        $self->sync_opennebula_hyps("-f");
```

Comment:
@stdweird maybe is better to include it into `@ONEHOST_SYNC_COMMAND` within commands.pm as default value:

> Readonly::Array my @ONEHOST_SYNC_COMMAND => qw(/usr/bin/onehost sync -f);

Instead include `-f` option as `cmd`, what do you think?